docker/app/Dockerfile

```diff
@@ -76,17 +76,18 @@ RUN apt-get update \
   && curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor -o /etc/apt/trusted.gpg.d/postgresql.gpg \
   && curl -fsSL https://deb.nodesource.com/setup_18.x | bash -s -- -y \
   && apt-get update \
-  && apt-get remove -y nodejs \
+  && apt-get remove -y nodejs || true \
```

Comment:
Sounds like you should just remove this line if node isn't installed by default.

Reply:
Node was, but maybe isn't now, I'll double check.  Thanks.